hadoop-ozone/freon/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyGenerator.java

```diff
@@ -646,6 +654,16 @@ long getUnsuccessfulValidationCount() {
     return validateWrites ? writeValidationFailureCount.get() : 0;
   }
 
+  /**
+   * Returns the current size of the buckets map.
+   *
+   * @return number of buckets created and added to the map
+   */
+  @VisibleForTesting
+  int getBucketMapSizeForTesting() {
```

Comment:
nit:
```suggestion
  int getBucketMapSize() {
```
No need to emphasise about "test" twice.
(but since ci is green, I'm ok to merge first)

Reply:
Yes please rename method @sreejasahithi.